luaui/Widgets/Include/AtlasOnDemand.lua

```diff
@@ -495,21 +495,21 @@ local function MakeAtlasOnDemand(config)
 	-- @param id the identifier of the item, needed for clearing its uv coordinate cache
 	function AtlasOnDemand:RemoveFromAtlas(uvcoords, id)
 		-- since it kinda has to by dynamic, it would be nice if we could mark some space as free on this
-		local xmin = math.floor(uvcoords.x * self.xslots)
-		local xmax =  math.ceil(uvcoords.X * self.xslots)
-		local ymin = math.floor(uvcoords.y * self.xslots)
-		local ymax =  math.ceil(uvcoords.Y * self.xslots)
+		local xmin = math.floor(uvcoords.x * self.xslots)+1
+		local xmax =  math.ceil(uvcoords.X * self.xslots)+1
+		local ymin = math.floor(uvcoords.y * self.yslots)+1
+		local ymax =  math.ceil(uvcoords.Y * self.yslots)+1
```

Comment:
why the +1? should both floor and ceil (min and max) have +1 and not -1 in one of them?

Reply:
Lua tables unfortunately work with indices starting from 1, not zero. The `{x,y}[min,max}` have a dual purpose here. They are both indices to Lua tables and multipliers for positions in the texture.

We can remove the `+1` here and add it to the Lua table accesses instead. I'm indifferent. Let me know which you prefer.

Reply:
That would be somebody else's call (@Beherith ?) but I added an inline comment in case the current version is to stay.